Remove references to Fabric from "Try kicking the Nginx machines" (#3449)
alphagov/govuk-developer-docs: source/manual/alerts/smokey-loop-tests.html.md

## Try kicking the Nginx machines

If many of the tests are failing in an AWS environment, it may be because the Nginx services haven't registered new

> boxes coming online or old ones going offline. You can try to restart the following services by:

Reviewer: ```suggestion
boxes coming online or old ones going offline. You can try to fix this by:
```

Previous wording was essentially "try restarting by...restarting (or reloading)".

SSHing into each `cache`, `draft_cache` and `monitoring` box and restarting nginx:

```bash
$ sudo service nginx reload
```

> Once that is complete you can restart the smokey loop on each monitoring box:

Reviewer: ```suggestion
Once that is complete you can restart the smokey loop on the monitoring box:
```

There should only be one of these.
